packages/prettier/package.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "@becklyn/prettier",
+    "version": "2.1.1",
+    "description": "The commonly used prettier configuration for projects at Becklyn",
+    "license": "BSD-3-Clause",
+    "homepage": "https://github.com/Becklyn/prettier",
+    "repository": {
+        "type": "git",
+        "url": "https://github.com/Becklyn/prettier.git"
+    },
+    "scripts": {
+        "test": "echo 'no tests defined' && exit 0",
+        "format": "prettier --write \"**/*.{ts,tsx,js,mjs,json,scss,css}\""
+    },
+    "main": "index.json",
+    "publishConfig": {
+        "access": "public"
+    },
```

Comment:
maybe you can compare the package json with the one of the other libs and clean this one up a little